### PR TITLE
Release v0.5. Also continue k2 rename to kraken-lib.

### PR DESCRIPTION
**Pull request by davidewatson**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -15,24 +15,45 @@ e2etester_version      = "0.2"
...
 
 jnlp_image             = "quay.io/${quay_org}/custom-jnlp:${custom_jnlp_version}"
-k2_tools_image         = "quay.io/${quay_org}/k2-tools:${k2_tools_image_tag}"
+kraken_tools_image     = "quay.io/${quay_org}/kraken-tools:${kraken_tools_image_tag}"
 ansible_lint           = "quay.io/${quay_org}/ansible-lint:latest"
 e2e_tester_image       = "quay.io/${quay_org}/e2etester:${e2etester_version}"
 docker_image           = "docker"
 
-podTemplate(label: 'k2', containers: [
+podTemplate(label: 'kraken_lib', containers: [
```
Comment (coffeepac) [inline Jenkinsfile]: Should be kraken-lib to match project name and node selector below.

File: Jenkinsfile [hunk 2]
```diff
@@ -134,16 +155,16 @@ podTemplate(label: 'k2', containers: [
         customContainer('docker') {
             // add a docker rmi/docker purge/etc.
             stage('Build') {
-                kubesh "docker rmi quay.io/${quay_org}/k2:k2-${env.JOB_BASE_NAME}-${env.BUILD_ID} || true"
-                kubesh "docker rmi quay.io/${quay_org}/k2:latest || true"
-                kubesh "docker build --no-cache --force-rm -t quay.io/${quay_org}/k2:k2-${env.JOB_BASE_NAME}-${env.BUILD_ID} docker/"
+                kubesh "docker rmi quay.io/${quay_org}/kraken-lib:kraken-${env.JOB_BASE_NAME}-${env.BUILD_ID} || true"
+                kubesh "docker rmi quay.io/${quay_org}/kraken-lib:latest || true"
+                kubesh "docker build --no-cache --force-rm -t quay.io/${quay_org}/kraken-lib:kraken-${env.JOB_BASE_NAME}-${env.BUILD_ID} docker/"
```
Comment (coffeepac) [inline Jenkinsfile]: For backwards compatibility we should be building the k2 and kraken-lib images for awhile.

File: Jenkinsfile [hunk 3]
```diff
@@ -134,16 +155,16 @@ podTemplate(label: 'k2', containers: [
...
             }
 
             //only push from master if we are on samsung-cnct fork
             stage('Publish') {
                 if (git_branch.contains(publish_branch) && git_uri.contains(github_org)) {
-                    kubesh "docker tag quay.io/${quay_org}/k2:k2-${env.JOB_BASE_NAME}-${env.BUILD_ID} quay.io/${quay_org}/k2:${image_tag}"
-                    kubesh "docker push quay.io/${quay_org}/k2:${image_tag}"
+                    kubesh "docker tag quay.io/${quay_org}/kraken-lib:kraken-${env.JOB_BASE_NAME}-${env.BUILD_ID} quay.io/${quay_org}/kraken-lib:${image_tag}"
```
Comment (coffeepac) [inline Jenkinsfile]: Again, build both k2 and kraken-lib for backwards compatibility.

**Comment by joejulian** [comment]
Still outstanding change requests in order to merge.

File: Jenkinsfile (hunk 2 above)
Comment (joejulian) [reply Jenkinsfile]: This still remains

File: Jenkinsfile (hunk 3 above)
Comment (joejulian) [reply Jenkinsfile]: As does this

**Comment by joejulian** [comment]
I'm an idiot. Never mind.

**Pull request merged**